app/javascript/packs/stylesheets/bootstrap.scss

```diff
@@ -31,6 +31,15 @@ body {
   border-radius: 0;
 }
 
+.icon-btn {
```

Comment:
are these specific to the history component? if so, maybe we should put it in a separate stylesheet for histories and give it a separate class

Reply:
right now they're just in the history component, however I made it generic in case we ever want to add these types of buttons elsewhere